### DISCUSSION TITLE
Not a user-friendly error message when saving text with Emoji

### DISCUSSION THREAD

**Discussion by dsheiko** [directus/directus#5634] [Feature Requests]
As an admin I type in a text field Emoji. When saving I get the system error message like on the attachment.

Expected behavior: Meaningful message like "You cannot use saving text with Emoji/extended UTF/whatever here"

Project details
Directus Version: 9.0.0-rc.64
Environment: production
OS: Ubuntu 18.04.2 LTS
Node: 11.15.1

![d301](https://user-images.githubusercontent.com/1393963/117952707-b3f82000-b315-11eb-971c-907e78091b2a.PNG)

**Comment by rijkvanzanten** [comment]
This is very database / configuration specific, and pretty difficult to reliably extract across the different database vendors 🤔 

For this particular issue I'm assuming you're using MySQL on a non-utf8mb4 collation, so it's more of a configuration problem then a bug in Directus we can solve for.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
